### PR TITLE
Eliminate unneeded impl classes (reopening #13)

### PR DESCRIPTION
**Pull request by eighthave**
I mistakenly closed #13 so this merge request is to just to remind us that it is actually open.  Please continue the discussion on #13

### CODE REVIEW

File: src/main/java/net/java/otr4j/session/SessionKeys.java [hunk 1]
```diff
@@ -1,54 +1,230 @@
+
```
Comment (cobratbq) [inline src/main/java/net/java/otr4j/session/SessionKeys.java:7]: Forgot to add/migratie otr4j license comment.

File: src/main/java/net/java/otr4j/session/SessionKeys.java [hunk 2]
```diff
@@ -1,54 +1,230 @@
...
-	public abstract DHPublicKey getRemoteKey();
-
-	public abstract KeyPair getLocalPair();
-
-}
\ No newline at end of file
+import net.java.otr4j.crypto.OtrCryptoEngine;
+import net.java.otr4j.io.SerializationUtils;
+
+public class SessionKeys {
```
Comment (cobratbq) [inline src/main/java/net/java/otr4j/session/SessionKeys.java:25]: Please migrate/add original "@author" comment of this class (it's in SessionKeysImpl)

**Comment by cobratbq** [comment]
I compared old files and the new merged file, just to be sure. Everything's in order as far as I can tell. Just 2 remarks for missing comments. Other than that everything's fine. I'll merge this once the comments are added.

**Comment by eighthave** [comment]
@cobratbq ok, fixed the missing license and author strings, this should be ready to go!

**Pull request merged**